proto/reaction.proto

```diff
@@ -209,6 +211,32 @@ message UnmeasuredAmount {
   string details = 2;
 }
 
+/**
+ * This qualitatively describes the apparent size and morphology of a Compound
+ * or a ReactionInput.
+ */
+message Texture {
+  // Note not all of the type options are mutually disjoint.
```

Comment:
this comment is true of all enums; please remove

Reply:
> this comment is true of all enums; please remove

removed in https://github.com/open-reaction-database/ord-schema/pull/698/commits/e0891b5b1a796cf295030bef6f72654f00ee5d25